pkg/util/docker/metadata.go

```diff
@@ -27,7 +27,7 @@ func getMetadata() (map[string]string, error) {
 		return metadata, err
 	}
 	// short timeout to minimize metadata collection time
-	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Why remove the `1*`, in the end it's the same but it makes the code less obvious in my opinion and don't bring anything.

Reply:
I actually wrote the `1*` in the first place so I'm fine putting it back if you think it's clearer 🙆‍♂️